reframe/core/buildsystems.py

```diff
@@ -236,7 +236,7 @@ class Make(BuildSystem):
     #:
     #: :type: :class:`str`
     #: :default: :class:`None`
-    makefile = variable(str, type(None), value=None)
+    makefile = variable(type(None), str, value=None)
```

Comment:
I will revert all these changes, since they are not needed now that the loader handles the `@none` special value.